saleor/account/tests/test_utils.py

```diff
@@ -189,3 +191,41 @@ def test_remove_the_oldest_user_address_if_address_limit_is_reached_limit_reache
 
     # then
     remove_the_oldest_user_address_mock.assert_called_with(customer_user)
+
+
+@pytest.fixture
+def users_with_similar_emails():
+    users = [
+        User.objects.create_user("andrew@example.com", "password"),
+        User.objects.create_user("Andrew@example.com", "password"),
+        User.objects.create_user("john@example.com", "password"),
+        User.objects.create_user("Susan@example.com", "password"),
+        User.objects.create_user("Cindy@example.com", "password"),
+        User.objects.create_user("CINDY@example.com", "password"),
```

Comment:
I'm wondering if we need to have passwords set for those users. If not, we can create the user instances in bulk instead of creating each user separately. 🤔

Reply:
You are right, we don't need password anymore, thanks!